notificationworkerlambda/cfn.yaml

```diff
@@ -134,7 +130,7 @@ Resources:
       Role: !GetAtt ExecutionRole.Arn
       Runtime: java8
       Timeout: 300
-      ReservedConcurrentExecutions: !Ref Concurrency
+      ReservedConcurrentExecutions: 100
```

Comment:
I might be getting the math wrong on this but:
maxConnections = LEAST({DBInstanceClassMemory/9531392},5000)
db.m4.large has 8gb ram
8000000000/9531392 = ~839

100 lambda executions * 20 fixedThreadPool = 2000 connections.
Will we have threadpool starvation: potentially lambdas starting that are unable to get any threads?